tutor/src/screens/teacher-dashboard/index.jsx

```diff
@@ -89,6 +89,8 @@ class TeacherDashboardWrapper extends React.Component {
     if (course.currentRole.isTeacherStudent && course.roles.teacher) {
       course.current_role_id = null;
     }
+    // getting the reference book data to show chapters and sections in the plan details
+    this.props.course.referenceBook.fetch();
   }
```

Comment:
probably should use `ensureLoaded` here as well since there's no reason to fetch it if it's already loaded.  The book contents never changes.